lib/clamp-device.in

```diff
@@ -73,7 +73,8 @@ LLD=$BINDIR/ld.lld
 
 
 HCC_BIN_PATH=$(dirname -- "$(readlink -f -- "$BASH_SOURCE")")
-ROCM_DEVICE_LIBS_SEARCH_PATHS="$HCC_BIN_PATH/../../rocdl/lib;$HCC_BIN_PATH/../rocdl/lib;@ROCM_DEVICE_LIB_PATHS@"
+ROCM_DEVICE_LIBS_SEARCH_PATHS="/opt/rocm/lib"
```

Comment:
Should it also check ROCM_DEVICE_LIB_PATHS? So like:
`ROCM_DEVICE_LIBS_SEARCH_PATHS="/opt/rocm/lib;@ROCM_DEVICE_LIB_PATHS@"`

Reply:
I think we also need to keep the search path relative (update needed) to HCC_BIN_PATH since some users may move the entire rocm directory to a non-default installation directory.
Also, I don't think we need to hardcode /opt/rocm/lib here

Reply:
Just to resolve this, ROCM_DEVICE_LIB_PATHS is set in the cmake file.  In this script, since we're relying on rocdl to already be installed, this script can't assume this variable has already been set.

Reply:
But we should use HCC_BIN_PATH.

Reply:
so, we do need ROCM_DEVICE_LIBS_PATH in clamp-device.in, for cmake-tests.  I've updated the files.